config/samples/azure_v1alpha1_appinsights.yaml

```diff
@@ -7,3 +7,7 @@ spec:
   location: westus
   resourceGroup: resourcegroup-azure-operators
   applicationType: other # Possible values include 'web' or 'other'
+
+  # Use the field below to optionally specify a different keyvault 
+  # to store the secrets in
+  #keyVaultToStoreSecrets: asoSecretKeyVault
```

Comment:
I had previously added the type, but not this to the sample YAML - another great catch!